app/views/jobseekers/registrations/confirm_destroy.html.slim

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title_prefix, t(".page_title")
+
+.govuk-main-wrapper
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      h1.govuk-heading-l = t(".page_title")
+
+      p = t(".page_description_1")
+      p = t(".page_description_2")
+
+      = form_for @close_account_feedback_form, url: registration_path(current_jobseeker), method: :delete do |f|
+        = f.govuk_error_summary
+
+        = f.govuk_radio_buttons_fieldset(:reason, legend: { size: "m" }) do
+          - Feedback.close_account_reasons.each_key do |reason|
+            - if reason == "other_close_account_reason"
```

Comment:
How does it know the reason in this conditional if the radio is rendered only if a particular reason has been selected?